Nos/Views/FullscreenProgressView.swift

```diff
@@ -34,6 +39,6 @@ struct FullscreenProgressView: View {
 
 struct FullscreenProgressView_Previews: PreviewProvider {
     static var previews: some View {
-        FullscreenProgressView(isPresented: .constant(true))
+        FullscreenProgressView(isPresented: .constant(true), text: nil)
```

Comment:
You can omit `text` here if you want. If you want to add a new preview that includes some text that might be nice also.

Reply:
[27b18b4](https://github.com/planetary-social/nos/pull/492/commits/27b18b4cf347b473ed12a0687fbb9679a6484e60)

Reply:
https://github.com/planetary-social/nos/assets/1935975/7448b815-9940-4b90-89b7-1474c6f21086